server/server.go

```diff
@@ -240,7 +234,7 @@ func (h RequestHandler) LoginRoute(res http.ResponseWriter, req *http.Request) {
 	loggedIn, _ := h.IsLoggedIn(req)
 	switch req.Method {
 	case "GET":
-		h.renderView(res, "login", TemplateData{LoginData{}, loggedIn, ""})
+		h.renderView(res, "login", TemplateData{Data: LoginData{}, LoggedIn: loggedIn, Title: ""})
```

Comment:
Might be nice to const all the view strings like `"login"` so refactoring is easier if you change or update this later?

Reply:
thanks for bringing it to attention! i'll bring it with me, and if it becomes a problem i'll probably revise the current approach with a set of enums or similar :~